### ISSUE TITLE
Not uploading to Evernote.

### ISSUE THREAD

**Issue by Patch1961** [yuki792/NeighborNote#8]
Hi,
I run Lubuntu version 14.04 and I have got NeighborNote to work in it. I am having issues with syncing with my Evernote account. NeighborNote seems to upload new notes into my Evernote account only once a day. I can write a note and upload it fine, but when I make changes to the note in Neighbornote, 10 minutes later, and do a sync, it doesn't upload the changes that I've made to the note. It'll download any changes that I've done on my Android phone fine, every time that I sync, but it won't upload.
Thanks.

**Comment by yuki792** [comment]
I'm very sorry for my late response. Recently, my life style has been changed drastically in a new environment. I haven't had enough time to take care of the project.
I will investigate about the issue as soon as possible. Please wait a little longer. If you have some additional information, please let me know.

Best regards,

**Comment by yuki792** [comment]
I think the issue is fixed now. But, I can't have confidence.
Could you check it by using following patch?

[https://drive.google.com/file/d/0B1VEmyWJSmlWal9uN1J3b2s4U2c/view?usp=sharing](https://drive.google.com/file/d/0B1VEmyWJSmlWal9uN1J3b2s4U2c/view?usp=sharing)

Please replace this patch with old neighbornote.jar in the installation directory.

Thanks,

**Comment by Patch1961** [comment]
Hi,
Yes the jar file seems to have solved the issue, thanks! Now
Neighbornote uploads & downloads every time.
The only thing left to fix is the carriage return bug. No matter whether
I have the carriage return fix in Debugging ticked or not ticked, the
carriage returns are still being added to my notes, when I upload them
and download them. If this were fixed I'd use Neighbornote every time.
Thanks for your efforts,
Patrick.

**Comment by yuki792** [comment]
Great!
I'll include this fix in next release. I close this ticket now.

Regarding the carriage return issue, could you open a new ticket?
And please tell me more details. I couldn't reproduce the issue.
